test/new/db/cmd/cmd_open

```diff
@@ -64,14 +64,14 @@ NAME=o-[0]
 FILE=malloc://1024
 EXPECT=<<EOF
 EOF
-CMDS=o-`o~[0]`;o
+CMDS=o- `o~[0]`;o
```

Comment:
Those syntax that accept a parameter attached to the command name are just weird and makes the splitting of the command name and the arguments hard! Let's just type an additional character and make sure most of the commands follow the syntax `<command-name> <argument1> <argument2> ...`

Reply:
o-1 should be a valid command as well as o- 1

Reply:
I know that *now* it is a valid command, but I think it's not consistent. When you are in bash, for example, `/bin/ls~` is not valid. `/bin/ls ~` is valid. I think we should stick to this way, where the command name is separated from the arguments.

Reply:
think about other use cases like f-* and such.. will thiss be a command or will need to be splitted as f- *. because that looks ugly to me. and i see this as a regression. i know r2 commands without spaces can be cryptic but they are so handy and easy to type.

Reply:
`f-*` is a command. But there's a difference:
- `f-*` is just `f-*`
- `o-`, on the other hand, could be `o-1` `o-2`, `o-12312321321`.

So, as you can see, `*` is just part of the command name, while the number in `o-` is just really an argument, it's not part of the cmd name